bv-java-grupa-1/iohomework/main/Main.java

```diff
@@ -0,0 +1,28 @@
+package main;
+
+import customexception.CountryNotFoundException;
+import dataprocessing.ProcessCountryData;
+
+import static utils.UserInput.getUserInput;
```

Comment:
It is a good exercise to practice the use of static imports; however, from a design point of view, is it necessary to create a separate class for this operation?